core-tests/src/test/java/org/mule/runtime/core/internal/transformer/simple/AbstractMessageTransformerTestCase.java

```diff
@@ -58,18 +72,37 @@ public DataType getReturnDataType() {
     return singletonMap("errorHandlerFromConfig", errorHandler);
   }
 
+  private MuleContext muleContextSpy;
+
+  @Before
+  public void before() {
+    muleContextSpy = spy(muleContext);
+    transformer.setMuleContext(muleContextSpy);
+  }
+
   @Test
   public void reusesDefaultFlowConstruct() throws MessageTransformerException {
-    MuleContext muleContextSpy = spy(muleContext);
-    transformer.setMuleContext(muleContextSpy);
     transform();
     transform();
     transform();
     verify(muleContextSpy, times(1)).getDefaultErrorHandler(any());
   }
 
+  @Test
+  public void doesntCreateTransformationEventIfEventProvided() throws MuleException {
```

Comment:
I would not use contractions in code names, but I've found other places where this is done.